app/assets/stylesheets/modern-business.scss

```diff
@@ -62,54 +62,52 @@ header.carousel .fill#slider-3 {
 /* Custom Styling */
 
 #candidates {
-  text-align: center
+  text-align: center;
 }
 
 #title-1 {
-  text-align: center;
   font-family: 'Quattrocento', serif;
-  font-size: 70px
-
+  font-size: 70px;
+  text-align: center;
 }
 
 #subtitle-1 {
-  text-align: center;
   font-family: 'Quattrocento', serif;
-  font-size: 30px
-
+  font-size: 30px;
+  text-align: center;
 }
 
 #copyright {
-  text-align: center
+  text-align: center;
 }
 
 #candidate-front-title {
-  text-align: center
+  text-align: center;
 }
 
 #issue-front-title {
-  text-align: center
+  text-align: center;
 }
 
 #button-front-page {
   padding-bottom: 40px;
-  padding-top: 60px
+  padding-top: 60px;
 }
 
 #header-into {
   text-align: center;
-  font-size: 18px
+  font-size: 18px;
 }
 
 /* Flash Notice Styling */
 
 #flash_notice {
-  background-color: #33CC33
+  background-color: #33CC33;
```

Comment:
Color literals like `#33CC33` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#33CC33` should be written as `#33cc33`